From fef37c7b0b0198e021fae1ec43859b59bd816120 Mon Sep 17 00:00:00 2001
From: Ola Jeppsson <ola@adapteva.com>
Date: Thu, 3 Sep 2015 01:02:02 +0200
Subject: [PATCH 25/33] epiphany: Add underscore prefix to init and fini

libgcc:
	* config/epiphany/crti.S: Add underscore prefix to init and
	fini functions so that we're still namespace clean.
	{USER_LABEL_PREFIX changed from "_" to "" in previous commit.}

Signed-off-by: Ola Jeppsson <ola@adapteva.com>
---
 libgcc/ChangeLog.epiphany     | 5 +++++
 libgcc/config/epiphany/crti.S | 8 ++++----
 2 files changed, 9 insertions(+), 4 deletions(-)
 create mode 100644 libgcc/ChangeLog.epiphany

diff --git a/libgcc/ChangeLog.epiphany b/libgcc/ChangeLog.epiphany
new file mode 100644
index 00000000000..20b77967a37
--- /dev/null
+++ b/libgcc/ChangeLog.epiphany
@@ -0,0 +1,5 @@
+2015-09-03  Ola Jeppsson  <ola@adapteva.com>
+	* config/epiphany/crti.S: Add underscore prefix to init and
+	fini functions so that we're still namespace clean.
+	{USER_LABEL_PREFIX changed from "_" to "" in previous commit.}
+
diff --git a/libgcc/config/epiphany/crti.S b/libgcc/config/epiphany/crti.S
index bc2ce196316..b9a71bc4a4b 100644
--- a/libgcc/config/epiphany/crti.S
+++ b/libgcc/config/epiphany/crti.S
@@ -22,13 +22,13 @@
 # <http://www.gnu.org/licenses/>.
 
 	.section .init
-	.global	init
+	.global	_init
 	.balign 2
-init:
+_init:
 	str lr,[sp],-4
 
 	.section .fini
-	.global	fini
+	.global	_fini
 	.balign 2
-fini:
+_fini:
 	str lr,[sp],-4
-- 
2.31.1

